tests/test_client.py

```diff
@@ -435,3 +436,24 @@ def test_files_import():
         file_url="https://static.scale.com/uploads/selfserve-sample-image.png",
         project_name=TEST_PROJECT_NAME,
     )
+
+
+current_timestamp = str(uuid.uuid4)[-9:]
+
+
+def test_list_teammates():
```

Comment:
@lautarorangil can you add a couple of teammates, and compare the lens to create some tests